app/resources/assets/sass/_home.scss

```diff
@@ -88,16 +90,16 @@ html > body #main > .home-page {
 	}
 	
 	.signin-signup {
-		
-		a {
+		display: flex;
```

Comment:
Similar to my comment on home.blade.php, what is this trying to do?  It looks like you're doing more than changing indentation but your commit messages don't explain it.  Is this intended to improve how the Sign In and Sign Up buttons look?